renderer/views/app.js

```diff
@@ -22,7 +22,8 @@ function App (state, dispatch) {
     state.playing.mouseStationarySince !== 0 &&
     new Date().getTime() - state.playing.mouseStationarySince > 2000 &&
     !state.playing.isPaused &&
-    state.playing.location === 'local'
+    state.playing.location === 'local' &&
+      state.playing.playbackRate === 0
```

Comment:
`playbackRate === 0`  means playback is paused, if you `&& !state.playing.isPaused` always gives you false?